e2e/nomostest/clusters/gke.go

```diff
@@ -222,16 +222,31 @@ func createGKECluster(t testing.NTB, name string) error {
 	// Standard-specific options (cannot specify for autopilot)
 	if !*e2e.GKEAutopilot {
 		if *e2e.GKEMachineType != "" {
-			args = append(args, "--machine-type", *e2e.GKEMachineType)
+			machineType := *e2e.GKEMachineType
+			// Override default value for stress tests
+			if *e2e.Stress && machineType == e2e.DefaultGKEMachineType {
```

Comment:
This could be a bit awkward if the user explicitly specifies the machine type which happens to be `DefaultGKEMachineType` while running stress tests

Reply:
I know. Couldn't think of another solution. Got a suggestion?

The --stress flag value isn't available at init time when the --machine-type default value is registered. And it needs to also fall back to the env var if not specified.

I could set the default to the empty string when configuring the flag, but then we have to process all the variables after parsing the flags. Since `go test` owns the command handler, including flag parsing, I don't know where to process the flags after parsing that won't cause race conditions when running tests in parallel. After init, the earliest code we have is nomostest.New, which runs in every test function.

Reply:
The only other solution I could come up with was to process the flags in every test, somewhere in nomostest.New, with a mutex or sync.Once. Seemed kinda overkill.

Reply:
I guess I could make a `TestMain(m *testing.M)` and parse flags there...

Reply:
Oh, hey, we have one already. I'll add it there.

Reply:
Done